lua/effects.lua

```diff
@@ -0,0 +1,17 @@
+-- EFFECT TO CHANGE LEVEL
+-- Use on [effect]
+-- apply_to=level
+-- increase=number or $formula
+-- set=number or $formula
+
+-- luacheck: push ignore 122
+function wesnoth.effects.level(unit, cfg)
```

Comment:
Considering loti is one of the big ones and not expected to be compatible with other addons its not that relevant, but I assume using wesnoth.effects.loti_level would also solve the warning.